R/add_stat.R

```diff
@@ -231,6 +233,7 @@ eval_fn_safe <- function(variable, tbl, fn) {
         data = tbl$inputs$data,
         variable = variable,
         by = tbl$inputs$by,
+        include = tbl$inputs$include,
```

Comment:
but I'm wondering if we should include tests similar to your `my_ttest()` examples for add_stat for tbl_summary objects, but I believe for the most part we would want access to the `include()` for the function formation, currently stuck/may need some assistance in this department